test/uk/gov/hmrc/pillar2stubs/controllers/TaxEnrolmentControllerSpec.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.pillar2stubs.controllers
 
 import org.scalatest.OptionValues
 import org.scalatest.freespec.AnyFreeSpec
-import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import org.scalatest.matchers.must.Matchers.mustEqual
```

Comment:
Appreciate that this isn't strictly related to the Scala 3 migration, but this indicates we're mixing should & must matchers in the same file. Could we please swap the two cases of `mustEqual` to `shouldEqual` in the file below, and remove this import?